apps/modernization-ui/src/apps/page-builder/pages/QuestionLibrary/QuestionLibraryTable.tsx

```diff
@@ -225,6 +235,7 @@ export const QuestionLibraryTable = ({ summaries, pages, qtnModalRef }: Props) =
                     handleSelected={handleSelected}
                     rangeSelector={true}
                     isLoading={isLoading}
+                    context={useContext(QuestionsContext)}
```

Comment:
We should use createContext and useContext and not need to pass through props:
https://react.dev/learn/passing-data-deeply-with-context